src/Trinity.Core/Storage/LocalMemoryStorage/LocalMemoryStorage.cs

```diff
@@ -81,7 +81,15 @@ private string WriteAheadLogFilePath
         static LocalMemoryStorage()
         {
             TrinityC.Init();
-            TrinityConfig.LoadTrinityConfig();
+            try
+            {
+                TrinityConfig.LoadTrinityConfig();
+            }
+            catch
+            {
+                Log.WriteLine(LogLevel.Error, "Failure to load config file, the default configuration takes effect");
+            }
+            CSynchronizeStorageRoot();
```

Comment:
CSynchronizeStorageRoot has return code. Please verify the sync result, and log errors if necessary.